src/controllers/class.js

```diff
@@ -0,0 +1,67 @@
+const mongoose = require('mongoose');
+const Class = require('../models/class');
+
+const updateClass = (req, res) => {
+  const { id } = req.params;
+  const {
+    day, hour, trainer, activity, slots,
+  } = req.body;
```

Comment:
Could you add an additional validation before the update? This is because the request body could be identical to an existing field in the database, potentially causing overwriting of data (two or more repeated instances). So, you should validate if the req.body contains identical data to any instance in the database.